src/Settings/Database/PersistentDatabaseAbstract.php

```diff
@@ -22,12 +20,23 @@ abstract class DatabaseAbstract extends SettingsDatabaseAbstract
      */
     protected string $password = '';
 
+    /**
+     * Which serializer strategy use by default
+     */
+    protected string $serializer = self::SERIALIZER_SERIALIZE;
+
+    public const SERIALIZER_SERIALIZE = 'serialize';
```

Comment:
Can we turn this into an enum pls

Reply:
I done it, but then realised, that users will need to use it in their eventHandlers and it can be confusing...
If you insist i will convert

Reply:
Done